spec/models/manageiq/providers/amazon/cloud_manager/metrics_capture_spec.rb

```diff
@@ -54,4 +54,23 @@
       end
     end
   end
+
+  context 'counters presence' do
```

Comment:
I would like to see also ```perf_capture('realtime')``` with a test if there is some actual value stored into the DB. There are several columns in Metric table, I would like to see which are being filled (we should have at least total_ram filled somewhere)